python/requirements.txt

```diff
@@ -9,3 +9,4 @@ convertdate>=2.1.2
 holidays>=0.9.5
 setuptools-git>=1.2
 python-dateutil>=2.8.0
+tqdm==4.42.1
```

Comment:
It'd be best to not pin the version in the requirements here, since some people may not have that level of control over the environment in which they are using the package.
4.42.1 is also a very recent release and so may not be available for all users.

This seems like pretty standard usage, so would this work with tqdm >= 4.36.1? That would be a better requirement on my end certainly.

Reply:
Sounds good will update!